tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -195,12 +195,37 @@ var CESIUM_MAP_VIEW = (function() {
                     if (curr_layer.options.params.VIEWPARAMS) {
                         parameters.VIEWPARAMS = curr_layer.options.params.VIEWPARAMS;
                     }
-
-                    var tile_wms = new Cesium.WebMapServiceImageryProvider({
-                        url: curr_layer.options.url,
-                        layers: curr_layer.options.params.LAYERS,
-                        parameters: parameters
-                    });
+                    if (curr_layer.times) {
+                        // times should be in this format "["20210322T112511Z", "20210322T122511Z", "20210323T032511Z"]"
```

Comment:
`// times should a JSON array string with times in ISO 8601 format: "["20210322T112511Z", "20210322T122511Z", "20210323T032511Z"]"`